RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionFlow+View.swift

```diff
@@ -84,7 +83,10 @@ extension DappInteractionFlow {
 					CaseLet(
 						/DappInteractionFlow.Path.MainState.oneTimePersonaData,
 						action: DappInteractionFlow.Path.Action.oneTimePersonaData,
-						then: { OneTimePersonaData.View(store: $0) }
+						then: {
+							OneTimePersonaData.View(store: $0)
+								.toolbar(.visible, for: .navigationBar)
```

Comment:
Any idea why it is needed here but not on the others?

Reply:
It's not directly related to this destination, the thing is that when this destination is shown **after showing the signing sheet**, then the navbar disappears. Presumably it would affect other destination shown after dismissing that sheet, but I am not sure there are any cases. For good measure I will move it to sit on `.destination` instead, so it always works. It doesn't hurt.